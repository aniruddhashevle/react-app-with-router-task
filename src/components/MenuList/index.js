import React from 'react';
import List from '@material-ui/core/List';
import ListItem from '@material-ui/core/ListItem';
import ListItemLink from '../ListItemLink';

const MenuList = ({ menuConfig = [] }) =>
    menuConfig && menuConfig.length &&
    <List className='menu-list'>
        {
            menuConfig.map(({
                id,
                title,
                subTitle,
                route,
                showDivider
            }) =>
                <ListItem divider={showDivider}>
                    <ListItemLink href={route} id={id} title={title}>
                        <h2 className="title">{title}</h2>
                        <p className="sub-title">{subTitle}</p>
                    </ListItemLink>
                </ListItem>

            )
        }
    </List>

export default MenuList
