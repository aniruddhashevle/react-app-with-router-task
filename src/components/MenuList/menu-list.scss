@import '../assets/styles/mixins.scss';
@import '../assets/styles/variables.scss';

.home-wrapper {
    .menu-list,
    .menu-list li { padding: 0; }
    .menu-list li {
        a {
            display: block;
            padding: 20px 0px;
            .title {
                margin: 0 0 20px 0;
                @include font-properties($font-size: 36px, $color: $brandTitleColor);
            }
            .description {
                margin: 0;
                @include font-properties($font-size: 20px, $color: $brandSubTitleColor);
            }
        }
    }
}