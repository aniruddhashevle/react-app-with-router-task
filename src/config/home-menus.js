export const HOME_MENUS = [
    {
        id: 1,
        title: 'Sales',
        subTitle: 'Insights about Sales',
        route: '/sales',
        component: 'Sales',
        showDivider: true
    },
    {
        id: 2,
        title: 'Customers',
        subTitle: 'Insights about Customers',
        route: '/customers',
        component: 'Customers',
        showDivider: true
    },
    {
        id: 3,
        title: 'Bulletin',
        subTitle: 'Daily summary of your business',
        route: '/#',
        component: null,
        showDivider: true
    },
    {
        id: 4,
        title: 'Ask',
        subTitle: 'Ask a question',
        route: '/#',
        component: null,
        showDivider: true
    },
    {
        id: 5,
        title: 'Task',
        subTitle: 'Tools for your business',
        route: '/#',
        component: null,
        showDivider: true
    },
    {
        id: 6,
        title: 'Connect',
        subTitle: 'Connect Us',
        route: '/#',
        component: null,
        showDivider: false
    },
]