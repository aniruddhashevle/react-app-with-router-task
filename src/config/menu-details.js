export const SALES_DETAILS = [
    {
        id: 1,
        title: 'Total Sales in November?',
        details: 'You have sales worth SGD 80,586.54 in last 30 days.',
        pinned: true,
    },
    {
        id: 2,
        title: 'Total Sales this week',
        details: 'You have not sold anything this week or you have not uploaded your invoices.',
        pinned: false,
    }
]

export const CUSTOMER_DETAILS = [
    {
        id: 1,
        title: 'Total Customers in November?',
        details: 'You have 30 customers in last 30 days.',
        pinned: true,
    },
    {
        id: 2,
        title: 'Total Customers this week',
        details: 'You have 5 customers in this week.',
        pinned: true,
    }
]