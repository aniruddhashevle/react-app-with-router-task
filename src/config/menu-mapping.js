import Sales from "../components/Sales";
import Customers from "../components/Customers";

const MENU_MAPPING = {};

MENU_MAPPING['Sales'] = Sales;
MENU_MAPPING['Customers'] = Customers;

export default MENU_MAPPING;