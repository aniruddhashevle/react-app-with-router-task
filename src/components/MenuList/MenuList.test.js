import React from 'react';
import MenuList from '../MenuList';
import { HOME_MENUS } from '../../config/home-menus';

it('MenuList renders correctly', () => {
    const wrapper = shallow(
        <MenuList menuConfig={HOME_MENUS} />
    )
    expect(wrapper).toMatchSnapshot();
});