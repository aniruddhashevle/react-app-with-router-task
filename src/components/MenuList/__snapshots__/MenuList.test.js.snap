// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuList renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MenuList
    menuConfig={
      Array [
        Object {
          "component": "Sales",
          "id": 1,
          "route": "/sales",
          "showDivider": true,
          "subTitle": "Insights about Sales",
          "title": "Sales",
        },
        Object {
          "component": "Customers",
          "id": 2,
          "route": "/customers",
          "showDivider": true,
          "subTitle": "Insights about Customers",
          "title": "Customers",
        },
        Object {
          "component": null,
          "id": 3,
          "route": "/#",
          "showDivider": true,
          "subTitle": "Daily summary of your business",
          "title": "Bulletin",
        },
        Object {
          "component": null,
          "id": 4,
          "route": "/#",
          "showDivider": true,
          "subTitle": "Ask a question",
          "title": "Ask",
        },
        Object {
          "component": null,
          "id": 5,
          "route": "/#",
          "showDivider": true,
          "subTitle": "Tools for your business",
          "title": "Task",
        },
        Object {
          "component": null,
          "id": 6,
          "route": "/#",
          "showDivider": false,
          "subTitle": "Connect Us",
          "title": "Connect",
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <WithStyles(ListItem)>
          <ListItemLink
            divider={true}
            href="/sales"
            title="Sales"
          >
            <h2
              className="title"
            >
              Sales
            </h2>
            <p
              className="description"
            >
              Insights about Sales
            </p>
          </ListItemLink>
        </WithStyles(ListItem)>,
        <WithStyles(ListItem)>
          <ListItemLink
            divider={true}
            href="/customers"
            title="Customers"
          >
            <h2
              className="title"
            >
              Customers
            </h2>
            <p
              className="description"
            >
              Insights about Customers
            </p>
          </ListItemLink>
        </WithStyles(ListItem)>,
        <WithStyles(ListItem)>
          <ListItemLink
            divider={true}
            href="/#"
            title="Bulletin"
          >
            <h2
              className="title"
            >
              Bulletin
            </h2>
            <p
              className="description"
            >
              Daily summary of your business
            </p>
          </ListItemLink>
        </WithStyles(ListItem)>,
        <WithStyles(ListItem)>
          <ListItemLink
            divider={true}
            href="/#"
            title="Ask"
          >
            <h2
              className="title"
            >
              Ask
            </h2>
            <p
              className="description"
            >
              Ask a question
            </p>
          </ListItemLink>
        </WithStyles(ListItem)>,
        <WithStyles(ListItem)>
          <ListItemLink
            divider={true}
            href="/#"
            title="Task"
          >
            <h2
              className="title"
            >
              Task
            </h2>
            <p
              className="description"
            >
              Tools for your business
            </p>
          </ListItemLink>
        </WithStyles(ListItem)>,
        <WithStyles(ListItem)>
          <ListItemLink
            divider={false}
            href="/#"
            title="Connect"
          >
            <h2
              className="title"
            >
              Connect
            </h2>
            <p
              className="description"
            >
              Connect Us
            </p>
          </ListItemLink>
        </WithStyles(ListItem)>,
      ],
      "className": "menu-list",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "1",
        "nodeType": "class",
        "props": Object {
          "children": <ListItemLink
            divider={true}
            href="/sales"
            title="Sales"
          >
            <h2
              className="title"
            >
              Sales
            </h2>
            <p
              className="description"
            >
              Insights about Sales
            </p>
          </ListItemLink>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <h2
                className="title"
              >
                Sales
              </h2>,
              <p
                className="description"
              >
                Insights about Sales
              </p>,
            ],
            "divider": true,
            "href": "/sales",
            "title": "Sales",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Sales",
                "className": "title",
              },
              "ref": null,
              "rendered": "Sales",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Insights about Sales",
                "className": "description",
              },
              "ref": null,
              "rendered": "Insights about Sales",
              "type": "p",
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "2",
        "nodeType": "class",
        "props": Object {
          "children": <ListItemLink
            divider={true}
            href="/customers"
            title="Customers"
          >
            <h2
              className="title"
            >
              Customers
            </h2>
            <p
              className="description"
            >
              Insights about Customers
            </p>
          </ListItemLink>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <h2
                className="title"
              >
                Customers
              </h2>,
              <p
                className="description"
              >
                Insights about Customers
              </p>,
            ],
            "divider": true,
            "href": "/customers",
            "title": "Customers",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Customers",
                "className": "title",
              },
              "ref": null,
              "rendered": "Customers",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Insights about Customers",
                "className": "description",
              },
              "ref": null,
              "rendered": "Insights about Customers",
              "type": "p",
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "3",
        "nodeType": "class",
        "props": Object {
          "children": <ListItemLink
            divider={true}
            href="/#"
            title="Bulletin"
          >
            <h2
              className="title"
            >
              Bulletin
            </h2>
            <p
              className="description"
            >
              Daily summary of your business
            </p>
          </ListItemLink>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <h2
                className="title"
              >
                Bulletin
              </h2>,
              <p
                className="description"
              >
                Daily summary of your business
              </p>,
            ],
            "divider": true,
            "href": "/#",
            "title": "Bulletin",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Bulletin",
                "className": "title",
              },
              "ref": null,
              "rendered": "Bulletin",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Daily summary of your business",
                "className": "description",
              },
              "ref": null,
              "rendered": "Daily summary of your business",
              "type": "p",
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "4",
        "nodeType": "class",
        "props": Object {
          "children": <ListItemLink
            divider={true}
            href="/#"
            title="Ask"
          >
            <h2
              className="title"
            >
              Ask
            </h2>
            <p
              className="description"
            >
              Ask a question
            </p>
          </ListItemLink>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <h2
                className="title"
              >
                Ask
              </h2>,
              <p
                className="description"
              >
                Ask a question
              </p>,
            ],
            "divider": true,
            "href": "/#",
            "title": "Ask",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Ask",
                "className": "title",
              },
              "ref": null,
              "rendered": "Ask",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Ask a question",
                "className": "description",
              },
              "ref": null,
              "rendered": "Ask a question",
              "type": "p",
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "5",
        "nodeType": "class",
        "props": Object {
          "children": <ListItemLink
            divider={true}
            href="/#"
            title="Task"
          >
            <h2
              className="title"
            >
              Task
            </h2>
            <p
              className="description"
            >
              Tools for your business
            </p>
          </ListItemLink>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <h2
                className="title"
              >
                Task
              </h2>,
              <p
                className="description"
              >
                Tools for your business
              </p>,
            ],
            "divider": true,
            "href": "/#",
            "title": "Task",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Task",
                "className": "title",
              },
              "ref": null,
              "rendered": "Task",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Tools for your business",
                "className": "description",
              },
              "ref": null,
              "rendered": "Tools for your business",
              "type": "p",
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "6",
        "nodeType": "class",
        "props": Object {
          "children": <ListItemLink
            divider={false}
            href="/#"
            title="Connect"
          >
            <h2
              className="title"
            >
              Connect
            </h2>
            <p
              className="description"
            >
              Connect Us
            </p>
          </ListItemLink>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <h2
                className="title"
              >
                Connect
              </h2>,
              <p
                className="description"
              >
                Connect Us
              </p>,
            ],
            "divider": false,
            "href": "/#",
            "title": "Connect",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Connect",
                "className": "title",
              },
              "ref": null,
              "rendered": "Connect",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Connect Us",
                "className": "description",
              },
              "ref": null,
              "rendered": "Connect Us",
              "type": "p",
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <WithStyles(ListItem)>
            <ListItemLink
              divider={true}
              href="/sales"
              title="Sales"
            >
              <h2
                className="title"
              >
                Sales
              </h2>
              <p
                className="description"
              >
                Insights about Sales
              </p>
            </ListItemLink>
          </WithStyles(ListItem)>,
          <WithStyles(ListItem)>
            <ListItemLink
              divider={true}
              href="/customers"
              title="Customers"
            >
              <h2
                className="title"
              >
                Customers
              </h2>
              <p
                className="description"
              >
                Insights about Customers
              </p>
            </ListItemLink>
          </WithStyles(ListItem)>,
          <WithStyles(ListItem)>
            <ListItemLink
              divider={true}
              href="/#"
              title="Bulletin"
            >
              <h2
                className="title"
              >
                Bulletin
              </h2>
              <p
                className="description"
              >
                Daily summary of your business
              </p>
            </ListItemLink>
          </WithStyles(ListItem)>,
          <WithStyles(ListItem)>
            <ListItemLink
              divider={true}
              href="/#"
              title="Ask"
            >
              <h2
                className="title"
              >
                Ask
              </h2>
              <p
                className="description"
              >
                Ask a question
              </p>
            </ListItemLink>
          </WithStyles(ListItem)>,
          <WithStyles(ListItem)>
            <ListItemLink
              divider={true}
              href="/#"
              title="Task"
            >
              <h2
                className="title"
              >
                Task
              </h2>
              <p
                className="description"
              >
                Tools for your business
              </p>
            </ListItemLink>
          </WithStyles(ListItem)>,
          <WithStyles(ListItem)>
            <ListItemLink
              divider={false}
              href="/#"
              title="Connect"
            >
              <h2
                className="title"
              >
                Connect
              </h2>
              <p
                className="description"
              >
                Connect Us
              </p>
            </ListItemLink>
          </WithStyles(ListItem)>,
        ],
        "className": "menu-list",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "1",
          "nodeType": "class",
          "props": Object {
            "children": <ListItemLink
              divider={true}
              href="/sales"
              title="Sales"
            >
              <h2
                className="title"
              >
                Sales
              </h2>
              <p
                className="description"
              >
                Insights about Sales
              </p>
            </ListItemLink>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <h2
                  className="title"
                >
                  Sales
                </h2>,
                <p
                  className="description"
                >
                  Insights about Sales
                </p>,
              ],
              "divider": true,
              "href": "/sales",
              "title": "Sales",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Sales",
                  "className": "title",
                },
                "ref": null,
                "rendered": "Sales",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Insights about Sales",
                  "className": "description",
                },
                "ref": null,
                "rendered": "Insights about Sales",
                "type": "p",
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "2",
          "nodeType": "class",
          "props": Object {
            "children": <ListItemLink
              divider={true}
              href="/customers"
              title="Customers"
            >
              <h2
                className="title"
              >
                Customers
              </h2>
              <p
                className="description"
              >
                Insights about Customers
              </p>
            </ListItemLink>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <h2
                  className="title"
                >
                  Customers
                </h2>,
                <p
                  className="description"
                >
                  Insights about Customers
                </p>,
              ],
              "divider": true,
              "href": "/customers",
              "title": "Customers",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Customers",
                  "className": "title",
                },
                "ref": null,
                "rendered": "Customers",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Insights about Customers",
                  "className": "description",
                },
                "ref": null,
                "rendered": "Insights about Customers",
                "type": "p",
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "3",
          "nodeType": "class",
          "props": Object {
            "children": <ListItemLink
              divider={true}
              href="/#"
              title="Bulletin"
            >
              <h2
                className="title"
              >
                Bulletin
              </h2>
              <p
                className="description"
              >
                Daily summary of your business
              </p>
            </ListItemLink>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <h2
                  className="title"
                >
                  Bulletin
                </h2>,
                <p
                  className="description"
                >
                  Daily summary of your business
                </p>,
              ],
              "divider": true,
              "href": "/#",
              "title": "Bulletin",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Bulletin",
                  "className": "title",
                },
                "ref": null,
                "rendered": "Bulletin",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Daily summary of your business",
                  "className": "description",
                },
                "ref": null,
                "rendered": "Daily summary of your business",
                "type": "p",
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "4",
          "nodeType": "class",
          "props": Object {
            "children": <ListItemLink
              divider={true}
              href="/#"
              title="Ask"
            >
              <h2
                className="title"
              >
                Ask
              </h2>
              <p
                className="description"
              >
                Ask a question
              </p>
            </ListItemLink>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <h2
                  className="title"
                >
                  Ask
                </h2>,
                <p
                  className="description"
                >
                  Ask a question
                </p>,
              ],
              "divider": true,
              "href": "/#",
              "title": "Ask",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Ask",
                  "className": "title",
                },
                "ref": null,
                "rendered": "Ask",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Ask a question",
                  "className": "description",
                },
                "ref": null,
                "rendered": "Ask a question",
                "type": "p",
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "5",
          "nodeType": "class",
          "props": Object {
            "children": <ListItemLink
              divider={true}
              href="/#"
              title="Task"
            >
              <h2
                className="title"
              >
                Task
              </h2>
              <p
                className="description"
              >
                Tools for your business
              </p>
            </ListItemLink>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <h2
                  className="title"
                >
                  Task
                </h2>,
                <p
                  className="description"
                >
                  Tools for your business
                </p>,
              ],
              "divider": true,
              "href": "/#",
              "title": "Task",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Task",
                  "className": "title",
                },
                "ref": null,
                "rendered": "Task",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Tools for your business",
                  "className": "description",
                },
                "ref": null,
                "rendered": "Tools for your business",
                "type": "p",
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "6",
          "nodeType": "class",
          "props": Object {
            "children": <ListItemLink
              divider={false}
              href="/#"
              title="Connect"
            >
              <h2
                className="title"
              >
                Connect
              </h2>
              <p
                className="description"
              >
                Connect Us
              </p>
            </ListItemLink>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <h2
                  className="title"
                >
                  Connect
                </h2>,
                <p
                  className="description"
                >
                  Connect Us
                </p>,
              ],
              "divider": false,
              "href": "/#",
              "title": "Connect",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Connect",
                  "className": "title",
                },
                "ref": null,
                "rendered": "Connect",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Connect Us",
                  "className": "description",
                },
                "ref": null,
                "rendered": "Connect Us",
                "type": "p",
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
